context-menu-copy-link-with-types/background.js

```diff
@@ -1,8 +1,12 @@
+// Callback clears runtime.lastError to prevent logspam errors from the menu
+// item being registered twice in case the background script is reloaded.
```

Comment:
```suggestion
// Callback reads runtime.lastError to prevent an unchecked error from being 
// logged when the extension attempt to register the already-registered menu 
//again. Menu registrations in event pages persist across extension restarts.
```